sdcm/nemesis.py

```diff
@@ -389,11 +389,16 @@ def _kill_scylla_daemon(self):
             # Let's wait for the target Node to have their services re-started
             self.log.info('Waiting scylla services to be restarted after we killed them...')
             self.target_node.wait_db_up(timeout=14400)
+            if self.cluster.params.get('use_mgmt'):
+                # Workaround for https://github.com/scylladb/scylla-manager/issues/2813
+                # When scylla take too long time to bring api port up
+                #  scylla-manager-agent fails to start and never go up
+                self.target_node.start_service('scylla-manager-agent', timeout=600, ignore_status=True)
             self.log.info('Waiting JMX services to be restarted after we killed them...')
             self.target_node.wait_jmx_up()
         self.cluster.wait_for_schema_agreement()
 
-    def disrupt_stop_wait_start_scylla_server(self, sleep_time=300):  # pylint: disable=invalid-name
+    def disrupt_sto4p_wait_start_scylla_server(self, sleep_time=300):  # pylint: disable=invalid-name
```

Comment:
you introduced a type in the function name

Reply:
fixed